projects/askama/build.sh

```diff
@@ -1,5 +1,5 @@
-#!/bin/bash -eu
-# Copyright 2023 Google LLC
+#!/bin/bash
+# Copyright 2024-2025 Google LLC
```

Comment:
Please leave it as 2023

Reply:
I removed the "-2025" addition.

The date comes from the moved file. I keep the change in two commits to make the providence more clear. Should I still back-date the file to 2023?

Reply:
ah right. Yeah, let's leave as is. I think that makes more sense.